plugin-server/tests/helpers/sql.ts

```diff
@@ -48,7 +48,6 @@ TRUNCATE TABLE
     posthog_event,
     posthog_pluginstorage,
     posthog_pluginattachment,
-    posthog_pluginlogentry,
```

Comment:
should we drop this table in a follow up PR?

Reply:
Hmm, normally it's not a great idea for us to drop tables, as users can be upgrading from a much older version of PostHog that could still be querying for it, but indeed in this case no ClickHouse-based installation should have _ever_ used this table anyway, so we should be able to do this.